lib/xboxkrnl/xboxkrnl.h

```diff
@@ -132,12 +133,12 @@ typedef enum _MODE {
 /**
  * This struct defines a counted string used for ANSI-strings
  */
-typedef struct _ANSI_STRING
+typedef struct
```

Comment:
Anonymous structs is a GCC extension, do we want to use those or should we name structs when we have the opportunity?

Reply:
This change also causes `warning: incompatible pointer types passing 'POBJECT_STRING'
      (aka 'STRING *') to parameter of type 'PANSI_STRING' (aka 'struct _STRING *') [-Wincompatible-pointer-types]`

So I'll have to look into fixing this in general.

Reply:
Fixed by reusing the `STRING` type.